configuration/analyzer_config.json

```diff
@@ -1165,5 +1165,17 @@
         "/opt/deploy/yara/stratosphere_rules/protocols"
       ]
     }
+  },
+  "ZoomEye": {
+    "type": "observable",
+    "observable_supported": ["ip", "domain"],
+    "requires_configuration": true,
+    "external_service": true,
+    "description": "Cyberspace Search Engine recording information of devices, websites, services and components etc..",
+    "python_module": "zoomeye.ZoomEye",
+    "additional_config_params": {
+      "api_key_name": "ZOOMEYE_KEY"
```

Comment:
please specify some default values here, making it like:
```
"additional_config_params": {
   "api_key_name": "ZOOMEYE_KEY",
   "page": 1,
   "facets": "app,os",
   "history: true
},
```

Reply:
I left that empty intentionally. Because, the more filters are present the lesser the response becomes. So, User defines if he wants specific output, Otherwise, It just gives all the output unfiltered. But, Now, I am adding default to `search_type` as `host`. Is it okay?

Reply:
```
"additional_config_params": {
   "api_key_name": "ZOOMEYE_KEY",
   "search_type": "host",
   "page": 1,
   "facets": "",
   "history: true
},
```

make it this. 

- The `page` = 1 is default anyway on zoomeye,
- `facets` is left as an empty string but hints the user that this config parameter does exist (very few users go through the docs in depth).
- `history` should be `true` by default. Make this change inside `ZoomEye.set_config` method too like: `additional_config_params.get("history", True)`